lib/helper/WebDriverIO.js

```diff
@@ -704,6 +706,29 @@ class WebDriverIO extends Helper {
   }
 
   /**
+   * Grab CSS property for given locator
+   *
+   * ```js
+   * I.grabCssPropertyFrom('h3', 'font-weight');
+   * ```
+   */
+  grabCssPropertyFrom(locator, cssProperty) {
```

Comment:
Same comment as for `seeCssPropertyOnElement`. 
This behavior is absolutely unpredictable in current implementation. `font-weight` of which element I receive when calling ` I.grabCssPropertyFrom('h3', 'font-weight');` ?